### PR TITLE
Bug 2033390: connect real MAPO images

### PR DESCRIPTION
**Pull request by iamemilio**
replaces stand in MAPO images with real ones

### CODE REVIEW

**Comment by iamemilio** [comment]
cc @mdbooth

File: install/0000_30_machine-api-operator_01_images.configmap.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ data:
       "clusterAPIControllerAWS": "registry.svc.ci.openshift.org/openshift:aws-machine-controllers",
       "clusterAPIControllerAlibaba": "registry.svc.ci.openshift.org/openshift:alibaba-machine-controllers",
       "clusterAPIControllerOpenStack": "registry.svc.ci.openshift.org/openshift:openstack-machine-controllers",
-      "clusterAPIControllerMAPO": "quay.io/shiftstack/machine-api-provider-openstack:latest",
+      "clusterAPIControllerMAPO": "quay.io/openshift/origin-machine-api-provider-openstack",
```
Comment (JoelSpeed) [inline install/0000_30_machine-api-operator_01_images.configmap.yaml:18]: Looking at the org, looks like this image hasn't actually been created yet, is the image mirroring in flight?

Comment (iamemilio) [reply install/0000_30_machine-api-operator_01_images.configmap.yaml:18]: yeah: https://github.com/openshift/release/pull/24933

Comment (mandre) [reply install/0000_30_machine-api-operator_01_images.configmap.yaml:18]: Is it expected the new image isn't in registry.svc.ci.openshift.org like the other ones on that list?

Comment (iamemilio) [reply install/0000_30_machine-api-operator_01_images.configmap.yaml:18]: according to mike, the correct repo is actually quay for all of them: https://github.com/openshift/machine-api-operator/pull/881

**Comment by JoelSpeed** [comment]
Based on the configurations I've seen, this should be accurate
/approve
/retest-required

**Pull request merged**